docs/common/ca.md

```diff
@@ -104,7 +104,64 @@ In addition to the above CAs, you can install other CAs via RPM. These only work
 Verifying CA Certificates
 -------------------------
 
-After installing the CA certificates, they can be verified with the following command:
+### Are up-to-date ###
```

Comment:
This line looks weird. How about making it an h2 with `## Verifying CA certificates are up-to-date`, and moving it below the `Post-installation` section? Perhaps also promote `Post-installation` to an h2 and change it to `## Verifying CA certificates are properly installed`?

Reply:
How about now?